doc/source/development/rfc/rfc105_safe_path_manipulation_functions.rst

```diff
@@ -0,0 +1,244 @@
+.. _rfc-105:
+
+===================================================================
+RFC 105: Add and use safe path manipulation functions
+===================================================================
+
+============== =============================================
+Author:        Even Rouault
+Contact:       even.rouault @ spatialys.com
+Started:       2025-01-11
+Status:        Draft
+Target:        GDAL 3.11
+============== =============================================
+
+Summary
+-------
+
+This RFC adds safe versions, for use by C++ code, of all functions of
+cpl_path.cpp (such as CPLGetPath(), CPLGetDirname(), CPLGetBasename(), CPLGetExtension(),
+CPLGetFormFilename(), CPLGetFormCIFilename(), etc.), that returns a result stored
```

Comment:
? should be CPLFormFilename() and CPLFormCIFilename() (no "Get")

Reply:
indeed, thanks! Fixed